From d9035d05351365873c2c4bb3c750b476ef644af0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@gmail.com>
Date: Wed, 22 May 2013 21:36:15 +0200
Subject: [PATCH] app: move display on client monitors with --full-screen

There used to be a check to fullscreen the only visible display on
current monitor, by checking the number of visible monitors. Now that
fullscreen is independant for each display, and goes on current monitor,
it's useless.

However, this code path is still used for the app --full-screen, at
startup time. And it is still nicer to open the display on respective
client monitors, rather than all on current monitor.
---
 src/virt-viewer-app.c | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/src/virt-viewer-app.c b/src/virt-viewer-app.c
index 539c4a4..c2ecc67 100644
--- a/src/virt-viewer-app.c
+++ b/src/virt-viewer-app.c
@@ -631,16 +631,12 @@ viewer_window_focus_out_cb(GtkWindow *window G_GNUC_UNUSED,
 }
 
 static void
-app_window_try_fullscreen(VirtViewerApp *self, VirtViewerWindow *win, gint nth)
+app_window_try_fullscreen(VirtViewerApp *self G_GNUC_UNUSED,
+                          VirtViewerWindow *win, gint nth)
 {
     GdkScreen *screen = gdk_screen_get_default();
-    gboolean move =
-        virt_viewer_app_get_n_windows_visible(self) > 1 ||
-        self->priv->fullscreen_auto_conf;
 
-    if (!move)
-        nth = -1;
-    else if (nth >= gdk_screen_get_n_monitors(screen)) {
+    if (nth >= gdk_screen_get_n_monitors(screen)) {
         DEBUG_LOG("skipping display %d", nth);
         return;
     }
